lib/pengine/bundle.c

```diff
@@ -63,17 +63,17 @@ allocate_ip(pe__bundle_variant_data_t *data, pe__bundle_replica_t *replica,
 
     data->ip_last = replica->ipaddr;
     switch (data->type) {
-        case PE_CONTAINER_TYPE_DOCKER:
-        case PE_CONTAINER_TYPE_PODMAN:
+        case PE_RUNTIME_DOCKER:
```

Comment:
isn't runtime rather something like runc, lxc, ...?

Reply:
Yeah, this may not be the best choice, but I couldn't find a more common name out in the wild to describe the idea -- I was hoping to avoid another pacemaker-specific term. I also liked that "runtime" was short :)

What do you think? Stick with "container type"? I wasn't completely happy with that because it's not really the type of the container itself, but perhaps it's more accurate than "runtime". Or maybe "launcher", or "container agent" since we're ultimately just using it as a resource agent name.

Reply:
didn't find a good name for the superclass of podman, docker, rocket, ...
what about container-toolset, container-control, container-admin ....
but of course container-launcher is possible as well ... all with or without the preceding container-

Reply:
container-infrastructure ... well terribly long ...

Reply:
I personally prefer "containter_launcher" :-)

Reply:
Indeed "container_agent" is also a good option for consistency.

Reply:
yes - good idea to name it seen from pacemaker pov as more general it might be a cmd-line-control-tool, actually a runtime directly, ...  or whatever the future will bring ...